### ISSUE TITLE
Installation instructions need updating

### ISSUE THREAD

**Issue by andrewmkiss** [NSLS-II/PyXRF#159]
I've had several users contact me with issues installing PyXRF. When creating a new install of pyxrf, there are new dependencies that are not installed by default using conda and need to be installed separately. I have updated instructions but this would be good to have on the website (or fixed in conda).

[Installing PyXRF.pdf](https://github.com/NSLS-II/PyXRF/files/3306209/Installing.PyXRF.pdf)

**Comment by mrakitin** [comment]
@andrewmkiss, could you please list those dependencies? They can be easily added to the conda package. Please note, that we only support Linux conda packages in our `lightsource2-tag` channel for now.

**Comment by andrewmkiss** [comment]
The additional dependencies I needed to install are cloudpickle and dask-core. Dask-core needs to be version 1.2.0.

Can you provide some clarification on what you mean by DAMA will only support the Linux conda packages? Almost all users are running either Windows or Mac at their home institutes. Is it possible to still push updates and bug fixes to these packages but only test them on Linux?

**Comment by mrakitin** [comment]
Thanks for the list of dependencies.

Sorry I wasn't precise regarding the packages. At the moment the limitation of our building system is that it only automatically builds packages for `linux-64` (and `noarch`) platforms. To enable conda builds of the packages for other platforms (OSX, Windows), we have to rework our approach on how we handle packaging. We have a summer project and an intern working on it. However, to provide you a quicker solution, @tacaswell just fixed the packages of interest via https://github.com/NSLS-II/lightsource2-recipes/pull/721. I will add the requested dependencies shortly.

**Comment by mrakitin** [comment]
@andrewmkiss, could you please explain how `cloudpickle` and `dask` appear to be the dependencies? I don't see any imports of those packages in the code base. Are you using some additions on top of `pyxrf`?

**Comment by andrewmkiss** [comment]
@mrakitin Hmmm, I'm not sure where they appeared. I got them impression that these were added so that pyxrf could directly interact with databroker, and remove the step to make_hdf() first. I'll double check on this. I'll confirm this is an issue with Linux/Windows installations.

Also, thank you for the explanation on the Windows/OSX platforms.

**Comment by mrakitin** [comment]
@andrewmkiss, thanks for the details. Databroker does have a requirement of `dask`:
https://github.com/bluesky/databroker/blob/bf9ed9805bd151dfc9a96c900a6d9ab0632f6cd3/requirements.txt#L3
however our recipe may be outdated:
https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/databroker/meta.yaml.

@tacaswell, @danielballan, should we update our databroker recipe with the `dask` requirement?

**Comment by danielballan** [comment]
It looks like the correct change would actually be to remove the dask requirement from `requirements.txt`, as it isn't actually used anywhere in the codebase.

```
$ git grep dask
doc/source/whats_new/v0.9.4.txt:* The HDF5 handlers have been refactored, and a new HDF5 handler returning dask
requirements.txt:dask
```

The line in the release notes for v0.9.4 shown there mentions some dask-based HDF5 handlers. Those were added but then reverted out, because the associated changes were too aggressive and broke some things.

**Comment by mrakitin** [comment]
@andrewmkiss ,`dask` dependency was removed from databroker via https://github.com/bluesky/databroker/pull/423, and should be available in [databroker v0.13.2](https://github.com/bluesky/databroker/releases/tag/v0.13.2).

**Comment by dmgav** [comment]
The instructions for installation of PyXRF from nsls2forge conda channel were updated as part of PR https://github.com/NSLS-II/PyXRF/pull/172 and https://github.com/NSLS-II/PyXRF/pull/173.
